lib/Macros/Sources/SwiftMacros/SwiftifyImportMacro.swift

```diff
@@ -45,19 +45,23 @@ protocol ParamInfo: CustomStringConvertible {
   ) -> BoundsCheckedThunkBuilder
 }
 
-func getParamName(_ param: FunctionParameterSyntax, _ paramIndex: Int) -> TokenSyntax {
-  let name = param.secondName ?? param.firstName
-  if name.trimmed.text == "_" {
-    return "_param\(raw: paramIndex)"
+func getParamName(_ param: FunctionParameterSyntax, _ paramIndex: Int, _ functionName: TokenSyntax) -> TokenSyntax {
+  if let secondName = param.secondName {
+    let paramName = secondName.trimmed.text
+    let funcName = functionName.trimmed.text
+    if paramName != funcName && "`\(paramName)`" != funcName {
+      return secondName
+    }
   }
-  return name
+  return "_param\(raw: paramIndex)"
```

Comment:
What happens if there is another param that happens to be named `_param1` or something like that?

A simpler scheme is to just mangle every parameter with `_\(funcName)param\(index)\(paramName)`. I _think_ that ensures no name clashes by construction. The downside is that it's ugly, but then you can opt into it only if there's any chance of a name clash.

Reply:
I guess including the function name guarantees not clashing with the function name. But the parameter name shouldn't *need* to be included to prevent clashing when all parameters are renamed.